tests/meson.build

```diff
@@ -11,7 +11,10 @@ test_src = files(
 test('all',
     executable(meson.project_name() + '_test',
         test_src,
-        dependencies : libgit4cpp_dep,
+        dependencies : [
+            dependency('catch2'),
```

Comment:
`catch2` could also be catch2v2. We should add a version number.

Probably `>>3.4.1` ?

Reply:
I have added `version: '>3.4.0'`. This is in line with the declaration in the `control` file.